financial-services-accelerator/components/org.wso2.financial.services.accelerator.consent.mgt.extensions/src/main/java/org/wso2/financial/services/accelerator/consent/mgt/extensions/internal/ConsentExtensionsServiceComponent.java

```diff
@@ -74,7 +75,8 @@ protected void deactivate(ComponentContext context) {
     }
 
     @Reference(
-            service = org.wso2.financial.services.accelerator.consent.mgt.service.ConsentCoreService.class,
+            name = "ConsentCoreService",
```

Comment:
```suggestion
            name = ConsentCoreService.class.getName(),
```

Reply:
Cannot use this as the name hence reverting back